.golangci.yml

```diff
@@ -99,6 +99,18 @@ linters-settings:
         deny:
           - pkg: "github.com/kumahq/kuma/app"
             desc: "pkg can't import app"
+      DenyGolangProtobuf:
+        files:
+          - $all
+        deny:
+          - pkg: github.com/golang/protobuf
+            desc: "github.com/golang/protobuf is deprecated; use google.golang.org/protobuf instead"
+      DenyGoogleProtoJson:
+        files:
+          - $all
+        deny:
+          - pkg: google.golang.org/protobuf/encoding/protojson
+            desc: "don't use the protojson package; use github.com/golang/protobuf/jsonpb instead"
```

Comment:
Could we add a note explaining why it shouldn't be used? Specifically, that it’s incompatible and might cause issues. We could include this as a comment, but it’s worth noting explicitly.